pkg/jms-sdk-go/model/account.go

```diff
@@ -55,6 +55,18 @@ type AccountDetail struct {
 	Privileged bool       `json:"privileged"`
 }
 
+type AssetChat struct {
+	ID   string `json:"id"`
+	Name string `json:"name"`
+}
+
+type AccountChatDetail struct {
+	ID       string    `json:"id"`
+	Name     string    `json:"name"`
+	Username string    `json:"username"`
+	Asset    AssetChat `json:"asset"`
+}
+
 type PermAccount struct {
 	Name       string  `json:"name"`
 	Username   string  `json:"username"`
```

Comment:
It seems that you have provided multiple structs (account chat detail and perm-account) without providing any context for the question asked. Please clarify what kind of details or changes you need to compare between existing structure and suggested structures, so I can help find discrepancies or suggest optimizations for your code.